src/vs/workbench/electron-browser/actions.ts

```diff
@@ -59,6 +59,31 @@ export class CloseEditorAction extends Action {
 	}
 }
 
+export class ForceCloseEditorAction extends Action {
```

Comment:
I have added a commit to move this into ```vs/workbench/browser/parts/editor/editorActions.ts```.

I have previously added it into ```vs/workbench/electron-browser/actions.ts```
because i found the command ```"workbench.action.closeActiveEditor"``` to be at 2 files (actions.ts & editorActions.ts) and it seemed like ```main.contribution.ts``` was importing the CloseEditorAction from the one in ```vs/workbench/electron-browser/actions.ts```.

It'd be nice if you could briefly explain the difference between the two files(actions.ts & editorActions.ts) with the same command ```"workbench.action.closeActiveEditor"```